insights/combiners/rhel_for_edge.py

```diff
@@ -17,34 +17,36 @@
 
 @combiner(InstalledRpms, CmdLine, ListUnits, RedhatRelease)
 class RhelForEdge(object):
-    """
-    Combiner for checking if the system is an edge computing systems.
-    Edge computing system packages are managed via rpm-ostree.
-    Red Hat CoreOS is also managed via rpm-ostree, use the string "Red Hat Enterprise Linux release"
-    from "/etc/redhat-release" to determine if it is an edge computing system as it is
-    "Red Hat Enterprise Linux CoreOS release" on RedHat CoreOs.
+    """Combiner for checking if the system is an edge computing system. Edge
+    computing as well as the Red Hat CoreOS packages are managed via rpm-ostree.
+    Use the string "Red Hat Enterprise Linux release" from
+    ``/etc/redhat-release`` to determine an edge computing system. The Red Hat
+    CoreOS system will have "Red Hat Enterprise Linux CoreOS release" as the
+    string..
 
     .. note::
         RHEL for EDGE is available and supported since RHEL 8.3.
 
-    When an edge computing system created from online console edge image is configured to use
-    automated management, the output of "rhc status" is the following::
+    When an edge computing system (created from online console edge image) is
+    configured to use the automated management, the output of ``rhc status`` is
+    as below::
 
-            Connection status for test.localhost:
+            Connection status for <HOST>:
             - Connected to Red Hat Subscription Management
             - The Red Hat connector daemon is active
 
-    If a system can upload insights archive, it must be connected to Red Hat Subscription Management,
```

Comment:
You omitted this part. I am unsure if it is important to have it here. What do you think?

Reply:
In my opinion the line was not "connecting" well with the subsequent line. Also it is obvious that the system should be subscribed & registered with Red Hat in order to upload an insights archive.

Reply:
That seems reasonable. I agree with you.